bluebottle/fsm/triggers.py

```diff
@@ -185,6 +186,8 @@ def _check_model_changed_triggers(self):
                         self._triggers.append(BoundTrigger(self, trigger))
 
     def execute_triggers(self, effects=None, **options):
+        if 'user' not in options and get_current_user():
+            options['user'] = get_current_user()
```

Comment:
Here's the nasty bit. @eodolphi do you think you have a better solution for this?